.github/workflows/c_release.yml

```diff
@@ -24,44 +24,47 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macOS-latest]
-#        os: [ubuntu-latest, macOS-latest, windows-latest]
         include:
           - os: ubuntu-latest
             output-name: sshnp-linux-x64
-            ext: ''
-            bundle: 'shell'
-          - os: macOS-latest
+            ext: ""
+            bundle: "shell"
+          - os: macos-13
             output-name: sshnp-macos-x64
-            ext: ''
-            bundle: 'shell'
-#          - os: macos-14
-#            output-name: sshnp-macos-arm64
-#            ext: ''
-#            bundle: 'shell'
-#          - os: windows-latest
-#            output-name: sshnp-windows-x64
-#            ext: '.exe'
-#            bundle: 'windows'
+            ext: ""
+            bundle: "shell"
+          - os: macos-14
+            output-name: sshnp-macos-arm64
+            ext: ""
+            bundle: "shell"
```

Comment:
I replaced macos-latest with macos-13 and macos-14 which are x64 and arm64 respectively.